numpy/core/_add_newdocs.py

```diff
@@ -5482,6 +5514,13 @@
     of these flags is in C-API documentation; they are largely useful
     for user-defined data-types.
 
+    Examples
+    --------
+
+    >>> x = np.dtype([('a', np.int32, 8), ('b', np.float64, 6)])
+    >>> x.flags
+    16
```

Comment:
Worth showing that this is equal to `np.core.multiarray.NEEDS_PYAPI`, perhaps by adding on the line below:
```
>>> np.core.multiarray.NEEDS_PYAPI
16
```

Reply:
@eric-wieser I think we should mention the correlation between the two if we are going to mention this. And I am not sure I understand that. Could you clarify this a little bit more?

Reply:
We already mention that correlation a few lines above